cfgmgr/intfmgr.cpp

```diff
@@ -626,6 +655,7 @@ bool IntfMgr::doIntfAddrTask(const vector<string>& keys,
 void IntfMgr::doTask(Consumer &consumer)
 {
     SWSS_LOG_ENTER();
+    static bool replayNotDone = true;
```

Comment:
can you please rename to "replayDone" and set to false.

Reply:
sure .. will change it